front/pages/api/w/[wId]/spaces/[spaceId]/index.ts

```diff
@@ -186,6 +186,16 @@ async function handler(
 
       if (name) {
         await space.updateName(auth, name);
+        // For regular spaces that only have a single group, update
+        // the group's name too (see https://github.com/dust-tt/tasks/issues/1738)
+        const singleGroup = space.groups[0];
```

Comment:
Let's have this in `space.updateName()` 🙏

Reply:
I thought about it before implementing; I thought it was cleaner outside, since theoretically groups and spaces may have different names even if they're linked. So for me choosing to update both should be explicit, not implicit inside space resource

Reply:
I see the updside of having it in space, though. So ok good for me